### DISCUSSION TITLE
dosemu2 for RHEL 9 / Rocky Linux

### DISCUSSION THREAD

**Discussion by ecasey** [dosemu2/dosemu2#2259] [General]
Is there any packaging of rpm packages for dosemu2 for RHEL9 ? Or a source rpm of dosemu2 ? that could be used with mock to generate an rpm for RHEL9? 

I've successfully built dosemu but would prefer to use the more maintained dosemu2 if available for RHEL9.

Thanks.

**Comment by stsp** [comment]
You can use `make rpm` to build rpm
locally. Or use this repo:
https://copr.fedorainfracloud.org/coprs/stsp/dosemu2/
